.github/workflows/ci.yaml

```diff
@@ -1,11 +1,17 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
 name: Tests
+
 on:
   pull_request:
     paths-ignore:
       - '.gitignore'
       - '.jujuignore'
       - 'LICENSE'
       - '**.md'
+  schedule:
+    - cron: '53 0 * * *' # Daily at 00:53 UTC
```

Comment:
Not an issue with the PR itself, but we may consider running the scheduled tests during slightly different times to not stampede on the runners.

Reply:
Starting all MySQL and PostgreSQL repos simultaneously at 00:53 might be noticeable in [Los Angeles](https://www.timeanddate.com/worldclock/converter.html?iso=20230217T005000&p1=1440&p2=137) (16:53 local time).